source/integrate-with-integration-environment/integrate-with-code-flow.html.md.erb

```diff
@@ -221,6 +221,10 @@ Location: https://YOUR_REDIRECT_URI?error=invalid_request
     <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">temporarily_unavailable</span></td>
     <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">If you're only making an authentication request (as opposed to requesting both authentication and identity), this error code means the GOV.UK Sign In authentication server is temporarily unavailable, which might be caused by temporary overloading or planned maintenance. </span><br><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">Make your authorisation request again in a few minutes. <br>  <br> If you're making an identity request and you get this error, it means the identity proofing and verification does not currently have capacity for this request.</span></td>
   </tr>
+  <tr>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">access_denied</span></td>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">The identity evidence provided by your user has a lower score than the identity confidence specified in your request. As a result, GOV.UK Sign In could not return the medium level of identity confidence (<code>Cl.Cm.P2</code>) and have instead returned a lower level of identity confidence. <br></br>To resolve this, make sure your user has entered the correct information for their identity evidence. If they have entered it correctly and this error is still appearing, you must ask your user to provide different identity evidence which meets a medium level of identity confidence.</span></td>
```

Comment:
```suggestion
    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">The identity evidence provided by your user has a lower score than the identity confidence specified in your request. As a result, GOV.UK Sign In could not return the medium level of identity confidence (<code>Cl.Cm.P2</code>) and instead returned a lower level of identity confidence. <br></br>To resolve this, make sure your user has entered the correct information for their identity evidence. If they have entered it correctly and this error is still appearing, you must ask your user to provide different identity evidence which meets a medium level of identity confidence.</span></td>
```
Sorry to nitpick, this little 'have' is still there! I'll give it an approval after this 😄

Reply:
no problem - it's now removed, thanks for the review